base/pkg/entry.jl

```diff
@@ -340,11 +344,14 @@ function update(branch::AbstractString)
             end
         end
         LibGit2.fetch(repo)
-        LibGit2.merge!(repo, fastforward=true) || LibGit2.rebase!(repo, "origin/$branch")
+        rebased = LibGit2.merge!(repo, fastforward=true)
```

Comment:
a fastforward is a rebase that squashes all intermediate commits, but I can change the variable name if that is confusing